source/resource/qx/tool/schema/compile-1-0-0.json

```diff
@@ -61,6 +61,10 @@
             "description": "The class name of the main application class (it typically inherits from `qx.application.Standalone` for web applications)",
             "type": "string"
           },
+          "workerClass" : {
+            "description": "EXPERIMENTAL: The class name of the worker application class, if there is one (it typically inherits from `qx.application.Basic`)",
+            "type": "string"
+          },
           "theme": {
```

Comment:
no; when you create a new instance of Worker(), you have to pass the filename of the script to load (or the actual javascript).  You can give it `__filename` so the same script loads (although that's for trivial examples and for Qooxdoo apps we would have to be careful to give the boot `index.js`) and it would then detect that it's loaded as a worker and act accordingly.

However, I thought that loading your normal / foreground app is not something you'd really want, and that the `qx.core.BaseInit` should decide what application class to load depending on whether running as a worker or the foreground thread.

This is something I'm experimenting with - it might actually be more appropriate to have an entirely different application in `compile.json`